frontend/dockerfile/dockerfile_test.go

```diff
@@ -235,7 +235,7 @@ func init() {
 
 	images := integration.UnixOrWindows(
 		[]string{"busybox:latest", "alpine:latest"},
-		[]string{"nanoserver:latest", "nanoserver:plus"})
+		[]string{"nanoserver:latest", "nanoserver:plus", "nanoserver:plus-busybox"})
```

Comment:
Are you sure these are needed? I don't see `busybox:stable` ? @daghack 

Maybe the test can be rewritten using different tagnames so that we don't need to pull more images just to check the tag string.